lib/sdr_client/deposit/file_type_file_set_strategy.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module SdrClient
+  module Deposit
+    # This strategy is for building the type of a fileset
+    class FileTypeFileSetStrategy
```

Comment:
Instead of multiple strategies that are hardcoded to a particular file set type, why not create a single strategy that is initialized with a file set type? Less code and in the future, if an additional file type is necessary, no need to change the client.

Reply:
I was trying to follow what we do in pre-assembly. This approach will allow us to make a "smart" strategy later if we need it without further changes.